cfme/test_framework/sprout/client.py

```diff
@@ -104,6 +104,11 @@ def provision_appliances(
             provider_type=None, lease_time=60, ram=None, cpu=None, **kwargs):
         # provisioning may take more time than it is expected in some cases
         wait_time = kwargs.pop('wait_time', 900)
+        try:
+            wait_time = int(wait_time)
+        except ValueError:
+            pass
```

Comment:
If the `ValueError` is hit, i.e. if `wait_time="10s"` or something, shouldn't we pass something else to `wait_for`? As `wait_for` will still try to use that string https://github.com/RedHatQE/wait_for/blob/e027318cd4202c5288246e12e591a86e3296115f/wait_for/__init__.py#L52 and then fail at https://github.com/RedHatQE/wait_for/blob/e027318cd4202c5288246e12e591a86e3296115f/wait_for/__init__.py#L176

Maybe would be best to add an update to `wait_for`?

Reply:
I agree I thought of that but it would be more difficult :)
I'm gonna update wait_for in a bit